rts/lra/test/basic/src/test/java/io/narayana/lra/arquillian/FailedLRAIT.java

```diff
@@ -298,4 +303,27 @@ private int removeFailedLRA(String recoveryUrl, String lra) {
             }
         }
     }
+
+    private void clearObjectStore() {
```

Comment:
I'm not sure if it's good idea to introduce a method which works with the file system based object store. What if we will be testing JDBC object store? There should be used some generic API instead, shouldn't be?

Reply:
I will remove the method